xcm/xcm-executor/src/config.rs

```diff
@@ -45,6 +45,10 @@ pub trait Config {
 	/// Combinations of (Asset, Location) pairs which we trust as teleporters.
 	type IsTeleporter: ContainsPair<MultiAsset, MultiLocation>;
 
+	/// A list of (Origin, Target) pairs allowing a given Origin to be substituted with its
```

Comment:
I guess I see where your confusion is -- you were thinking that the concatenation of `origin` and `target` here would be the foreign account's ML, but this is not true; the origin _already_ contains the account ID junction.